### PR TITLE
Fix: incorrect quotes and apostrophes

### PR DESCRIPTION
**Pull request by Gootector**
Fix: incorrect quotes and apostrophes.

Greetings,
Gootector

Resolves: #NNNNN 

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/project/internal/projectactionscontroller.cpp [hunk 1]
```diff
@@ -378,7 +378,7 @@ bool ProjectActionsController::closeOpenedProject(bool quitApp)
 
 IInteractive::Button ProjectActionsController::askAboutSavingScore(INotationProjectPtr project)
 {
-    std::string title = qtrc("project", "Do you want to save changes to the score “%1” before closing?")
+    std::string title = qtrc("project", "Do you want to save changes to the score "%1" before closing?")
```
Comment (Jojo-Schmitz) [inline src/project/internal/projectactionscontroller.cpp:381]: Needs to be
std::string title = qtrc("project", "Do you want to save changes to the score \"%1\" before closing?")

File: src/project/internal/projectactionscontroller.cpp [hunk 2]
```diff
@@ -1203,7 +1203,7 @@ bool ProjectActionsController::askIfUserAgreesToOpenProjectWithIncompatibleVersi
 
 bool ProjectActionsController::askIfUserAgreesToOpenCorruptedProject(const String& projectName, const std::string& errorText)
 {
-    std::string title = mtrc("project", "File “%1” is corrupted").arg(projectName).toStdString();
+    std::string title = mtrc("project", "File "%1" is corrupted").arg(projectName).toStdString();
```
Comment (Jojo-Schmitz) [inline src/project/internal/projectactionscontroller.cpp:1206]: Needs to be
`std::string title = mtrc("project", "File \"%1\" is corrupted").arg(projectName).toStdString();`

File: src/project/internal/projectactionscontroller.cpp [hunk 3]
```diff
@@ -1218,7 +1218,7 @@ bool ProjectActionsController::askIfUserAgreesToOpenCorruptedProject(const Strin
 
 void ProjectActionsController::warnProjectCannotBeOpened(const String& projectName, const std::string& errorText)
 {
-    std::string title = mtrc("project", "File “%1” is corrupted and cannot be opened").arg(projectName).toStdString();
+    std::string title = mtrc("project", "File "%1" is corrupted and cannot be opened").arg(projectName).toStdString();
```
Comment (Jojo-Schmitz) [inline src/project/internal/projectactionscontroller.cpp:1221]: Needs to be
`std::string title = mtrc("project", "File `\"%1\" is corrupted and cannot be opened").arg(projectName).toStdString();`

**Comment by Jojo-Schmitz** [comment]
the `"` inside string need to get 'escaped' by a `\`

File: src/project/internal/projectactionscontroller.cpp (hunk 1 above)
Comment (Gootector) [reply src/project/internal/projectactionscontroller.cpp]: Done.

File: src/project/internal/projectactionscontroller.cpp (hunk 2 above)
Comment (Gootector) [reply src/project/internal/projectactionscontroller.cpp]: Done.

File: src/project/internal/projectactionscontroller.cpp (hunk 3 above)
Comment (Gootector) [reply src/project/internal/projectactionscontroller.cpp]: Done.

**Comment by Gootector** [comment]
@Jojo-Schmitz See here: https://github.com/musescore/MuseScore/pull/17748/files.

**Comment by Jojo-Schmitz** [comment]
> @Jojo-Schmitz See here: https://github.com/musescore/MuseScore/pull/17748/files.

That's quite many commits...

**Comment by Gootector** [comment]
> > @Jojo-Schmitz See here: https://github.com/musescore/MuseScore/pull/17748/files.
> 
> That's quite many commits... and with intelligible commit messages

No one merged to the master 😞

**Comment by Jojo-Schmitz** [comment]
Not yet...
Better squash all these commit with that identical message

**Comment by Gootector** [comment]
> Not yet... Better squash all these commit with that identical message

Squash? 😄

**Comment by Jojo-Schmitz** [comment]
Yup, that is the git term for this

**Comment by Gootector** [comment]
> Yup, that is the git term for this

Yes, I found: https://www.git-tower.com/learn/git/faq/git-squash. How I can do it using Microsoft Edge...

**Comment by Jojo-Schmitz** [comment]
Not with Egge at all. Just with git itself

**Comment by Gootector** [comment]
> Not with Egge at all. Just with git itself

So, I can't do it. I'm using MS Edge for PRs.

**Comment by cbjeukendrup** [comment]
Don't worry about squashing, we can do it when merging. 

But I'm not sure if we want to do this. I believe that for user-visible text, those “curly quotes” are _better_ than "straight quotes". @bkunda what do you think?

**Comment by Gootector** [comment]
> Don't worry about squashing, we can do it when merging.
> 
> But I'm not sure if we want to do this. I believe that for user-visible text, those “curly quotes” are _better_ than "straight quotes". @bkunda what do you think?

' and " are known as neutral quotation marks, and in my opinion should be used in the code. Anyone can change these signs in own language using translations on Transifex.

**Comment by Jojo-Schmitz** [comment]
FWIW: my opinion too. Also it is all or nothing, for consistency we should not use a mix of curly and straight quotes

**Comment by bkunda** [comment]
As far as the UI is concerned, I would prefer it that we stick to curly quotes. 
This is the generally accepted norm in good typographic practice. 
I have no opinion about what quotation marks appear in the code.

**Comment by Jojo-Schmitz** [comment]
I take that back, it isn't that much, just 10 string using 'normal' quotes.
But different languages use differen quotes, in e.g. German we use „...1“, not `“...”`

**Comment by Jojo-Schmitz** [comment]
But very many using normal single quutes

**Comment by Jojo-Schmitz** [comment]
So far the vast majority of single quites are straight ones, this PR fixed those that are not, and I think it should, due to the amount of new strinbs that would otherwise get introduced.
For double quotes this is a bit different, 10 times straight vs. 3 times curly, still a majority, but not unmanageble many

**Comment by Gootector** [comment]
And? Close or not?

**Comment by cbjeukendrup** [comment]
My opinion:
- The strings as written in the code represent the "base language", which is US English. Thus, since in US English curly quotes are preferred, the code should use those curl quotes as well.
- This will _not_ prevent translators from using whatever quotes they want in the translation. 
- If we'd write straight quotes in the code, then someone will need to go to Transifex and "translate" _every_ instance of them into curly quotes. This is very inefficient.

So, we should fix the code to be consistent and have curly quotes everywhere. But this should not be done for 4.1, because we're too close to the release for that. 

Therefore, I think we'll have to close this PR, and create a new one in the future, to _properly_ fix the inconsistency by using curly quotes everywhere.

**Pull request closed without merging**